src/main/java/com/redhat/jenkins/plugins/ci/CIMessageNotifier.java

```diff
@@ -89,30 +115,13 @@ public String getMessageContent() {
     public void setMessageContent(String messageContent) {
         this.messageContent = messageContent;
     }
-    public boolean isFailOnError() {
+    public boolean getFailOnError() {
```

Comment:
why the change from is() to get()? usually a boolean is an is()

Reply:
I will change this back.